### ISSUE TITLE
Ex3. Motion not right in high speed

### ISSUE THREAD

**Issue by kaikai23** [CMM-22/a2#29]
Hello, my robot looks all right when moving speed is under 1.8m/s. But when I give a high speed like 2.5m/s, it suddenly becomes crazy. 

I don't know why this happens because everything is fine when speed is normal, and I have also passed EX.4 but stick to numerical Jacobian in IK_solver. Do you have any insight where might the problem be? Thanks!

https://user-images.githubusercontent.com/71677542/160912592-46360c18-5c7f-49f2-84e8-6ba9f9a4c69c.mov

**Comment by Marialenakdg** [comment]
I have the same problem! And no idea what's going on

**Comment by eastskykang** [comment]
If a commanded velocity is too high, the feet targets are too far away from current feet position, that consequently leads to singularity (where there's no solution to reach the targets) This may cause the problem. 

Can you tell me what happens if you reduce the command velocity after this happens? If the behavior goes back to normal, then you are on the right track.

**Comment by kaikai23** [comment]
Yes, after reducing the velocity, the robot behaves back to normal. Thank you!
